pages/linux/aspell.md

```diff
@@ -6,7 +6,7 @@
 
 `aspell check {{path/to/file}}`
 
-- List mispelled words from standard input:
+- List misspelled words from standard input:
```

Comment:
Lol :P

Reply:
Was that typo a joke? In a page all about finding typos, there was a typo, and the typoed word was `misspelled`!! 🤣